### ISSUE TITLE
Adopt Jas when Married to Shane

### ISSUE THREAD

**Issue by Kibbleknight** [StardewModders/mod-ideas#605]
After you've been married to Shane for a little while, and Jas is at Max Hearts, an event where Shane asks you if you'd be willing to adopt Jas alongside him will occur when waking up one morning. You can say yes or no.

-say "Yes" and Shane will ask if she should move in with the Player or stay at Marnie's Ranch. 
--"Move In" Shane will then ask Jas how she feels about it, and then she will move into the farmhouse with the Player and Shane. Possibly give her an "Add-On" room similar to the spouse rooms when she does. Her schedule will be adjusted slightly for her being on the farm instead of at the ranch. 
--"Stay at Marnie's" Jas will be happy about having parents again, but will not be moved from the ranch house.

-say "No" and Shane will ask why. You can say "Not ready for that yet" and he'll ask you again in the future. Shane will still adopt her himself, and she'll refer to him as Dad or as Shane interchangeably.

Basically you just adopt her, but instead of using the base game really sparse child system, she is an npc. You can still have kids with Shane normally, Jas will talk about being happy about being a big sister, and Refer to you as "Mother/Father" or "Mommy/Daddy." And You can also hug her similarly to how you can hug Krobus.

**Comment by LenneDalben** [comment]
Isn't this similar to #513?

**Comment by Pathoschild** [comment]
Yep; I copied the suggestion to that ticket.
